Donut/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
```

Comment:
Cool, let's try this out then. I'll keep the the blueprint init files (e.g. `Donut/modules/example/__init__.py`) the same, since we'll always only need `flask.Blueprint`.